### PR TITLE
feat: support use of zepter for feature checks

### PR DESCRIPTION
**Pull request by dandanlen**
# Pull Request

Closes: PRO-1474

Zepter is a tool that helps make sure cargo features are applied/propagated correctly. 

This PR just contains the default zepter config. 

To run it, you need to install zepter:

`cargo install zepter -f --locked`

Then `zepter check` to check and `zepter` to fix the issues.

For now I didn't include the applied changes - we will apply them *after* the next polkadot sdk update to avoid additional merge conflicts.

### CODE REVIEW

File: .zepter.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# Then `zepter check` to check the workspace and `zepter` to fix the issues.
+
+version:
+  # File format for parsing it:
+  format: 1
+  # Minimum version of the binary that is expected to work. This is just for printing a nice error
+  # message when someone tries to use an older version.
+  binary: 0.13.2
+
+# The examples in this file assume crate `A` to have a dependency on crate `B`.
```
Comment (kylezs) [inline .zepter.yaml:15]: what examples?

**Review by kylezs — APPROVED** [review]

File: .zepter.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - ["format", "features", "--offline", "--locked", "--quiet"]
+  # Same as `check`, but actually fix the issues instead of just reporting them:
+  default:
+    - [$check.0, "--fix"]
+    - [$check.1, "--fix"]
+
+# Will be displayed when any workflow fails:
+help:
+  text: |
+    It looks like one more more Zetper feature checks failed; please check the console output.
```
Comment (dandanlen) [inline .zepter.yaml:48]:
```suggestion
    It looks like one more more Zepter feature checks failed; please check the console output.
```

File: .zepter.yaml (hunk 1 above)
Comment (dandanlen) [reply .zepter.yaml:15]: I didn't write this (it's from the polkadot repo) but it's in reference to the comments below mentioning A and B.

Comment (dandanlen) [reply .zepter.yaml:15]:
```suggestion
# The explanatory comments in this file assume crate `A` to have a dependency on crate `B`.
```

**Pull request merged**